bin/run

```diff
@@ -32,11 +33,21 @@ parser.add_argument(
 parser.add_argument(
     "--model_url", type=str, default='', help="Model's public URL")
 parser.add_argument(
-    '--input_file', type=str, default="", help='load the data from a numpy file format (.npy)')
+    '--input_file', type=str, default="", help='Load the data from a numpy file format (.npy)')
+parser.add_argument(
+    '--data_dir', type=str, default="", help='Data directory with TFRecords file only')
 parser.add_argument(
     '--plaintext', action='store_true', help='Run the model in plaintext')
 parser.add_argument(
-    '--epochs', type=int, default=10, help='Number of times to run a prediction')
+    '--iterations', type=int, default=10, help='Number of times to run a prediction')
+parser.add_argument(
+    '--data_prep_path', type=str, default='', help='Path to a saved model with data pre-processing')
+parser.add_argument(
+    '--data_prep_start_node', type=str, default='serialized_example:0', help='Select data pre-processing start' \
```

Comment:
Should this have a default? I’m legitimately not sure

Reply:
good point. this default are more specific to t2t. It's just a bit annoying to specify it every time we run a quick experiment locally. But i could remove it. It makes more sense in general.

Reply:
let's keep it in for now